.gitignore

```diff
@@ -35,6 +35,9 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+data/OpSim_database/
```

Comment:
@Nikki1510 do you have data sets in this folder that are needed for the tests? If so, these are being ignored to be committed. Perhaps this is the reason for the test failures?